jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/thread/Invocable.java

```diff
@@ -13,6 +13,8 @@
 
 package org.eclipse.jetty.util.thread;
 
+import org.eclipse.jetty.util.Callback;
+
```

Comment:
Unused import?

Reply:
Is used in javadoc, but I have made that fully qualified instead of an import.